core/src/main/java/org/mule/config/pool/StoppableRunnable.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.config.pool;
+
+
+public abstract class StoppableRunnable implements Runnable
```

Comment:
Class-level javadoc.  Also it's more `cancel` than `stop` isn't it? If possible compose Runnable and another interface if one exists.